daal4py/sklearn/test/test_patching/test_patching.py

```diff
@@ -0,0 +1,220 @@
+    'LogisticRegressionCV float32 predict_log_proba',
+    'LogisticRegressionCV float32 score',
+    'LogisticRegressionCV float64 decision_function',
+    'LogisticRegressionCV float64 predict',
+    'LogisticRegressionCV float64 predict_proba',
+    'LogisticRegressionCV float64 predict_log_proba',
+    'LogisticRegressionCV float64 score',
+    'LogisticRegressionCV uint8 decision_function',
+    'LogisticRegressionCV uint8 predict',
+    'LogisticRegressionCV uint8 predict_proba',
+    'LogisticRegressionCV uint8 predict_log_proba',
+    'LogisticRegressionCV uint8 score',
+    'LogisticRegressionCV uint16 decision_function',
+    'LogisticRegressionCV uint16 predict',
+    'LogisticRegressionCV uint16 predict_proba',
+    'LogisticRegressionCV uint16 predict_log_proba',
+    'LogisticRegressionCV uint16 score',
+    'LogisticRegressionCV uint32 decision_function',
+    'LogisticRegressionCV uint32 predict',
+    'LogisticRegressionCV uint32 predict_proba',
+    'LogisticRegressionCV uint32 predict_log_proba',
+    'LogisticRegressionCV uint32 score',
+    'LogisticRegressionCV uint64 decision_function',
+    'LogisticRegressionCV uint64 predict',
+    'LogisticRegressionCV uint64 predict_proba',
+    'LogisticRegressionCV uint64 predict_log_proba',
+    'LogisticRegressionCV uint64 score',
+    # --------------- Scikit ---------------
+    'Ridge float16 predict',
+    'Ridge float16 score',
+    'RandomForestClassifier int8 predict_proba',
+    'RandomForestClassifier int8 predict_log_proba',
+    'RandomForestClassifier int16 predict_proba',
+    'RandomForestClassifier int16 predict_log_proba',
+    'RandomForestClassifier int32 predict_proba',
+    'RandomForestClassifier int32 predict_log_proba',
+    'RandomForestClassifier int64 predict_proba',
+    'RandomForestClassifier int64 predict_log_proba',
+    'RandomForestClassifier float16 predict_proba',
+    'RandomForestClassifier float16 predict_log_proba',
+    'RandomForestClassifier float32 predict_proba',
+    'RandomForestClassifier float32 predict_log_proba',
+    'RandomForestClassifier float64 predict_proba',
+    'RandomForestClassifier float64 predict_log_proba',
+    'RandomForestClassifier uint8 predict_proba',
+    'RandomForestClassifier uint8 predict_log_proba',
+    'RandomForestClassifier uint16 predict_proba',
+    'RandomForestClassifier uint16 predict_log_proba',
+    'RandomForestClassifier uint32 predict_proba',
+    'RandomForestClassifier uint32 predict_log_proba',
+    'RandomForestClassifier uint64 predict_proba',
+    'RandomForestClassifier uint64 predict_log_proba',
+]
+
+if __name__ == '__main__':
+    import subprocess
+    process = 'IDP_SKLEARN_VERBOSE=INFO ' + \
+        'python daal4py/sklearn/test/test_patching/launch_algorithms.py > ' + \
+        'daal4py/sklearn/test/test_patching/raw_log'
+    p1 = subprocess.Popen(process, shell=True)
+    subprocess.Popen.wait(p1)
+    fin = open('daal4py/sklearn/test/test_patching/raw_log', 'r')
```

Comment:
I think this is unnecessary. you can get the result from the stdout stream. for example:
```python
p = Popen(cmd, stdout=PIPE, stderr=PIPE)
stdout, stderr = p.communicate()
```